app/jobs/after_transition_tasks_for_rejected_return_job.rb

```diff
@@ -16,17 +16,17 @@ def perform(submission, transition)
           submission.transition_to!(:resubmitted, {auto_resubmitted: true})
         end
       end
-      message_class = message_class_for_state(submission.current_state)
-      if message_class
-        ClientMessagingService.send_system_message_to_all_opted_in_contact_methods(
-          client: submission.client,
-          message: message_class,
-          locale: submission.client.intake.locale
-        )
-      end
+      # message_class = message_class_for_state(submission.current_state)
```

Comment:
these messages wouldn't get sent anyways because we don't have clients on our submission but better to comment it out